packages/utils/src/lib/mock/schema-helper.mock.ts

```diff
@@ -47,14 +47,14 @@ export function mockPluginConfig(opt?: {
       command: 'node',
       args: [
         '-e',
-        `require('fs').writeFileSync('${outputPath}', '${JSON.stringify(
+        `require('fs').writeFileSync('${outputDir}', '${JSON.stringify(
           audits.map(
             ({ slug }) =>
               mockAuditReport({ auditSlug: slug }) satisfies AuditReport,
           ),
         )}')`,
       ],
-      outputPath: outputPath,
+      outputFile: outputDir,
     },
```

Comment:
Changing this variable to `outputDir` is misleading, should either be `outputFile` or stay as `outputPath`.

Reply:
nice catch. renamed to outputFile